clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -52,7 +54,9 @@
 #include "llvm/ADT/STLFunctionalExtras.h"
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Casting.h"
 #include "llvm/Support/Debug.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
nit: These two look unrelated to changes in this file (although could be not unused).